modules/ROOT/pages/transaction-management.adoc

```diff
@@ -4,69 +4,100 @@ include::_attributes.adoc[]
+* `NOT_SUPPORTED`: Execute outside any existent transaction. Operations that support transactions provide this setting. Scopes that support transactions provide this setting.
 
-.XML Example: Transaction Configuration in the Message Source
+== Configuring a Transaction in the Message Source
+
+You can start a transaction from a message source. In this case, the entire flow becomes a transaction. This is useful when working with messaging connectors to prevent the consumption of the message if a problem occurs when processing it, allowing you to retry later (because of the rollback).
+
+You can configure a transaction in a message source from its *Advanced* tab in Studio:
+
+image::transaction-config-studio-vm.png[transaction-config-studio-vm]
+
+The XML example below illustrates a `vm:listener` message source configured to initiate a transaction:
 
 [source,xml,linenums]
 ----
 include::{examplesdir}/transaction-management_2.xml[]
 ----
 
-In Studio, operations in these connectors provide an Advanced tab for setting up transactions:
-
-* JMS:
- ** Listener supports Local and XA transaction types. When set, the entire flow is treated as a transaction.
- ** Consume and Publish operations support transactions.
-* VM:
- ** Listener supports Local and XA transaction types. When set, the entire flow is treated as a transaction.
- ** Consume and Publish operations support transactions.
-* Database:
- ** All database operations support transactions.
-
-Note that a Mule flow can begin with a non-transactional connector (such as HTTP) that requires a transaction within the flow. For example, a Mule flow might accept information from an external Web service, then transform the data before charging a credit card and saving invoice information to a database. In such a situation, you might use the Try scope to set up a transaction by wrapping the credit card charge and database commit operations within a transaction to ensure either complete success or complete failure and rollback.
+== Configuring a Transaction in a Try Scope
 
-== Transactional Actions
+A Mule flow can also begin with a non-transactional connector (such as HTTP) that requires a transaction within the flow. For example, a Mule flow might accept information from an external Web service, then transform the data before charging a credit card and saving invoice information to a database. In such a situation, you use the Try scope to set up a transaction by wrapping the credit card charge and database commit operations within a transaction to ensure either complete success or complete failure and rollback.
 
-A Transactional Action (`transactionalAction`)
+You can initiate a transaction from a Try scope component either by dragging it from the Mule Palette in Studio to your flow, or including the `try` element in your configuration XML.
```

Comment:
While I'm reading this, I find it confusing. It should be something like  
"You can initiate a transaction from a Try scope component:
. Add the Try Scope to your flow, either by dragging it from the Mule Palette in Studio to your flow, or including the `try` element in your configuration XML.
. Configure the Transaction Type and the Transaction Action from the *General* tab in studio, or including the `transactionalAction` and `transactionType` attributes in your configuration XML. "